src/Components/Auction/AuctionRegistrationModal.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react"
+
+import { Box, Button, Flex, Serif } from "@artsy/palette"
+import { ConditionsOfSaleCheckbox } from "Components/Auction/ConditionsOfSaleCheckbox"
+import { Modal } from "Components/Modal/Modal"
```

Comment:
@erikdstock You should switch this to the palette modal as we don't want to use the reaction one any more. Let me know if you still are seeing issues but it should be good if you rebase Palette.

Reply:
done